python/gast_to_code/py_gast_to_code_converter.py

```diff
@@ -50,14 +50,28 @@ def handle_for_range(gast, lvl=0):
         start = str(start_value)
 
         # incrementor
-        incrementor_value = gast["update"]["right"]["value"]
         incrementor_op = gast["update"]["op"]
-        if incrementor_op == "-=":
-            incrementor = "-" + str(incrementor_value)
-        elif incrementor_op == "+=":
-            incrementor = str(incrementor_value)
+
+        # Normal aug assign expression
+        if "right" in gast["update"]:
+            incrementor_value = gast["update"]["right"]["value"]
+            if incrementor_op == "-=":
+                incrementor = "-" + str(incrementor_value)
+            elif incrementor_op == "+=":
+                incrementor = str(incrementor_value)
+            else:
+                incrementor = "unsupported update expression"
+        
+        # ++ or -- expression
         else:
-            incrementor = "unsupported update operation"
+            if incrementor_op == "++":
```

Comment:
Will this convert `a++` --> `a += 1`?

Reply:
Currently for Python, we have it translating to `a++` to `++a`